.circleci/config.yml

```diff
@@ -558,21 +557,21 @@ jobs:
           name: Publish Docker images
           command: |
             export DOCKER_CLI_EXPERIMENTAL=enabled
-            docker tag "grafana/grafana:master-${CIRCLE_SHA1}" "grafana/grafana:master-new-pipeline"
-            docker push "grafana/grafana:master-new-pipeline"
-            docker tag "grafana/grafana-arm32v7-linux:master-${CIRCLE_SHA1}" "grafana/grafana-arm32v7-linux:master-new-pipeline"
-            docker push "grafana/grafana-arm32v7-linux:master-new-pipeline"
-            docker tag "grafana/grafana-arm64v8-linux:master-${CIRCLE_SHA1}" "grafana/grafana-arm64v8-linux:master-new-pipeline"
-            docker push "grafana/grafana-arm64v8-linux:master-new-pipeline"
+            docker tag "grafana/grafana:master-${CIRCLE_SHA1}" "grafana/grafana:master"
+            docker push "grafana/grafana:master"
+            docker tag "grafana/grafana-arm32v7-linux:master-${CIRCLE_SHA1}" "grafana/grafana-arm32v7-linux:master"
+            docker push "grafana/grafana-arm32v7-linux:master"
+            docker tag "grafana/grafana-arm64v8-linux:master-${CIRCLE_SHA1}" "grafana/grafana-arm64v8-linux:master"
+            docker push "grafana/grafana-arm64v8-linux:master"
       - run:
           name: Publish Docker manifest
           command: |
             export DOCKER_CLI_EXPERIMENTAL=enabled
-            docker manifest create "grafana/grafana:master-new-pipeline" \
-              "grafana/grafana:master-new-pipeline" \
-              "grafana/grafana-arm32v7-linux:master-new-pipeline" \
-              "grafana/grafana-arm64v8-linux:master-new-pipeline"
-            docker manifest push "grafana/grafana:master-new-pipeline"
+            docker manifest create "grafana/grafana:master" \
+	      "grafana/grafana:master" \
```

Comment:
The workflow is failing because of indentation of this line.

Reply:
@papagian Ah oops, thanks.